examples/tests/sdk_tests/website/website.w

```diff
@@ -0,0 +1,24 @@
+bring cloud;
+
+let w = new cloud.Website(path: "./website");
+let config = { json: 1 };
+
+class Util {
+    extern "../external/http.js" static inflight http(url: str): Json;
+    extern "../external/fs.js" static readFile(path: str): str;    
+  }
+ 
+let indexFile = Util.readFile("./website/website/index.html");
+let otherFile = Util.readFile("./website/website/inner-folder/other.html");
+
+w.addJson("config.json", config);
+
+assert(w.path.endsWith("sdk_tests/website/website"));
```

Comment:
can you add a comment explaining what's being tested here (it seems like we are verifying that `path` is resolved to an absolute path, correct?)